plugins/services/src/js/components/ConfigurationMapAction.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import classNames from "classnames";
+
+const ConfigurationMapAction = ({ isHover, onClick, children }) => {
+  const classes = classNames("button button-link flush", {
+    "table-display-on-row-hover": isHover
+  });
+
+  return (
+    <a className={classes} onClick={onClick}>
```

Comment:
@ahoskins I have a few questions because this component is effectively a <Button/> without all the props available on a <Button/> and with an isHover. This makes me curious about it's necessity. Are their any drawbacks or blockers for removing this entirely and just rendering the `children` directly?
e.g. instead of:
```
 <ConfigurationMapAction
    onClick={onEditClick.bind(this, tabViewID)}
    isHover={true}
  >
    Edit
  </ConfigurationMapAction>
```
we just do this:
```
  <button
    onClick={onEditClick.bind(this, tabViewID)}
    classNames="button button-link flush table-display-on-row-hover"
  >
    Edit
  </button>
```

Reply:
I think since this component is always used beside the other components for configuration map, it should also be a component. ex:

```
<ConfigurationMapRow>
  <ConfigurationMapLabel>...
  <ConfigurationMapValue>...
  <ConfigurationMapAction> ...
</ConfigurationMapRow>
```

Agree?

But I do think that if a component, this should be able to render either a `<button>` or an `<a>`. There will be some usages when we want to pass it an `onClick` to execute some javascript, but other cases when we give it an `href`. For example in the node licensing rows, one of actions will be opening a link "Upgrade" that directs to a support.mesosphere.com.

Maybe this is too complex, and it's easier to just remove the component and render HTML tags directly. Thoughts @dcos/frontend-foundation